RELEASING.md

```diff
@@ -0,0 +1,49 @@
+# Release Process
+
+## Versioning
+The versioning follows the process described at [semver.org](https://semver.org/).
+
+### Public API
+We consider the public API of this project to be whatever is included in the
+`LIBRARY_PUBLIC_HEADERS` defined in `src/CMakeLists.txt`. Currently, this includes
+all the header files in `src/mococrw/`. Changes to the files will be versioned according
+to semantic versioning scheme.
+
+## PRs & Releasing
+
+### PR Checklist
+(probably to be extended)
+
+What should be done with every PR (if no new release is created):
+* CHANGELOG is updated with the details of the change (Unreleased section)
```

Comment:
Rather with a summary of the change than the details? Nobody cares about (implementation) details.

Reply:
Additionally: CI testing against project specific CI systems.